config.yml

```diff
@@ -4,4 +4,9 @@ in:
   max: 1000
 out:
   type: postgresql
-  mode: direct_merge
+  mode: insert_direct
+  create_table_option: |
+    ;CREATE TABLE new_table (id BIGSERIAL PRIMARY KEY, like testing2 including all);
+    DROP TABLE <replace_with_table_name>;
```

Comment:
So it is basically to drop the current table and wait the new table to be renamed to the old table name, right?
With this, can we guarantee the app won't error because table doesn't exist for a while? @akhyar93

Reply:
Yes, that's correct mas @absyah 
I've been experimenting with several configurations. and looks like this one is giving the correct order. so this SQL is running directly after the table creation and before the data from BQ inserted into our table.